modules/weko-workflow/weko_workflow/utils.py

```diff
@@ -253,8 +272,61 @@ def validation_item_property(mapping_data, identifier_type, properties):
     :return: error_list or None
     """
     error_list = {'required': [], 'pattern': [], 'pmid': '',
-                  'doi': '', 'url': ''}
+                  'doi': '', 'url': '', 'either': []}
+    empty_list = deepcopy(error_list)
+
+    if properties.get('required'):
+        error_list_required = validattion_item_property_required(
+            mapping_data, identifier_type, properties['required'])
+        if error_list_required:
+            error_list['required'] += error_list_required['required']
+            error_list['pattern'] += error_list_required['pattern']
+
+    if properties.get('either'):
+        error_list_either = validattion_item_property_either_required(
+            mapping_data, identifier_type, properties['either'])
+        if error_list_either:
+            error_list['either'] += error_list_either
+
+    if error_list == empty_list:
+        return None
+    else:
+        error_list['required'] = list(set(error_list['required']))
```

Comment:
Just curious, Is it only `error_list['required']` that needs to be converted? Should we also convert for  `pattern `and `either` in order to prevent it in the future?